iot/http_example/cloudiot.rest.json

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
What is this file for?

Reply:
Removed, the device API was to be exposed as Apiary via device REST discovery.